docs/connection-dialog.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Build the Connection Dialog
+---
+
+The connection dialog prompts the user to enter connection and authentication information. That information is passed into the Connector Builder script to build the connection string. The dialog appears when creating a new connection or editing an existing connection and is used by both Tableau Desktop and Tableau Server.
+
+The connection dialog can be defined in two different ways:
+- [Connection Dialog v1]({{ site.baseurl }}/docs/ui) 
+- [Connection Dialog v2]({{ site.baseurl }}/docs/mcd) in Tableau 2020.3 or later 
+
+Connection Dialog v2 is the recommended pattern for new connectors.
+
+## Set connector name and vendor information
+
+The connector is displayed as "[Display Name] by [Company Name]" in the connection dialog and connection list.
+
+"For support, contact [Company Name]" is displayed at the bottom left of the connector. Clicking this link sends the user to the support link defined in the manifest. This link also displays in error messages. The support link must use HTTPS to be packaged into a TACO file.
+
+These elements are defined in the manifest.xml file:
+```xml
+<connector-plugin class='postgres_odbc' superclass='odbc' plugin-version='0.0.0' name='PostgreSQL ODBC' version='20.1'>
+  <vendor-information>
+      <company name="Company Name"/>
+      <support-link url = "http://example.com"/>
+  </vendor-information>
+  ...
+</connector-plugin>
```

Comment:
Missing the driver download link, I'm pretty sure that is shown when trying to launch a connector where the driver is not installed nowadays.